app/main.py

```diff
@@ -1,3 +1,82 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: Union[Distance, int, float]) -> Distance:
```

Reply:
use `annotations from __futures__`

Reply:
Oh, cool idea, because I didn't understand how to fix the annotation of the output, python indicated that there was an error
Now I understand
thank you